src/server/client.ts

```diff
@@ -528,7 +528,29 @@ namespace ts.server {
         }
 
         getDocumentHighlights(fileName: string, position: number): DocumentHighlights[] {
-            throw new Error("Not Implemented Yet.");
+            var lineOffset = this.positionToOneBasedLineOffset(fileName, position);

```

Comment:
If you destructured this into `line` and `offset` you could use shorthand properties below